tests/python_functional/functional_tests/source_drivers/file_source/test_acceptance.py

```diff
@@ -24,8 +24,8 @@
 
 import pytest
 
-input_log = "<38>Feb 11 21:27:22 {} testprogram[9999]: test message\n".format(socket.gethostname())
-expected_log = "Feb 11 21:27:22 {} testprogram[9999]: test message\n".format(socket.gethostname())
+input_log = "<38>Feb 11 21:27:22 {} testprogram[9999]: test message\n".format(socket.gethostname().split('.')[0])
+expected_log = "Feb 11 21:27:22 {} testprogram[9999]: test message\n".format(socket.gethostname().split('.')[0])
 
```

Comment:
this should probably be extracted into a function (e.g. getshorthostname() or something)

Reply:
@smortex First of all thanks for running and fixing tests.
I agreed with @bazsi , it would be better to create a function for this. I would put it under something like this: `src/common/network_operations.py`
And in this case I would create separate functions like: 
* get_hostname(), 
* get_short_hostname(), 
* get_fqdn()